desktop/libs/Project/.flowconfig

```diff
@@ -94,4 +94,4 @@ suppress_comment=\\(.\\|\n\\)*\\$FlowIssue\\((\\(>=0\\.\\(2[0-5]\\|1[0-9]\\|[0-9
 suppress_comment=\\(.\\|\n\\)*\\$FlowFixedInNextDeploy
 
 [version]
-^0.25.0
+^0.32.0
```

Comment:
Pretty sure there's other stuff in this file that needs to change too between RN versions. But we can try it like this and update subsequently